### ISSUE TITLE
Sentry.init is not a function

### ISSUE THREAD

**Issue by franklinjavier** [GoogleChrome/lighthouse#13100]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://noop.com

### What happened?
I updated the lighthouse cli with the below code
```sh
$ npm i -g lighthouse
```

When I tried to run, i'm getting this error: 
```sh
$ lighthouse "https://www.belezanaweb.com.br"
TypeError: Sentry.init is not a function
    at begin (file:///usr/local/lib/node_modules/lighthouse/lighthouse-cli/bin.js:131:12)%

$ lighthouse --version
unknown

```

### What did you expect?

N/a

### How were you running Lighthouse?

CLI

### Lighthouse Version

unknown

**Comment by franklinjavier** [comment]
I

> I have fixed this at #13101, and you also could fix it by editing the file "/node_modules/lighthouse/lighthouse-cli/bin.js":
> 
> ```diff
> - import * as Sentry from '../lighthouse-core/lib/sentry.js';
> + import Sentry from '../lighthouse-core/lib/sentry.js';
> ```

I’m passing `--enable-error-reporting` to resolve now

**Comment by franklinjavier** [comment]
https://github.com/GoogleChrome/lighthouse/pull/13104
